tests/pytests/test_missing.py

```diff
@@ -481,3 +481,33 @@ def testMissing(env):
 
         # Test missing fields indexing on JSON documents
         JSONMissingTest(env, conn)
+
+
+def testMissingWithExists():
+    """Tests the missing values indexing feature with the `exists` operator"""
+
+    env = Env(moduleArgs="DEFAULT_DIALECT 2")
+    conn = getConnectionByEnv(env)
+
+    # Create an index with a TAG field that indexes missing values
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 'foo', 'TEXT', 'INDEXMISSING').ok()
+
+    # Add some documents, with\without the indexed fields.
+    conn.execute_command('HSET', 'doc1', 'foo', 'hello')
+    conn.execute_command('HSET', 'doc2', 'goo', 'world')
+
+    # Search for the documents with the indexed fields (sanity)
+    ismissing = env.cmd('FT.SEARCH', 'idx', 'ismissing(@foo)')
+    exists = env.cmd('FT.AGGREGATE', 'idx', '*', 'LOAD',  '2', 'foo', 'goo', 'FILTER', '!EXISTS(@foo)')
```

Comment:
Let's add a test for the case missing values are used in the filter expression (i.e., the same with a different function, instead of `EXISTS()`.

Reply:
Also, just for completion, add the positive case.

Reply:
See `test_aggregate_filter_on_missing_values`
I think the positive case is probably checked in test_aggregate.py as well.